docs/topics/js/js-debugging.md

```diff
@@ -13,7 +13,7 @@ on it. Refer to your browser's documentation for more details.
 
 To debug Kotlin/JS in the browser:
 
-1. Run the project by calling one of the available `*Run` Gradle tasks, for example, `browserDevelopmentRun` or
+1. Run the project by calling one of the available `run` Gradle tasks, for example, `browserDevelopmentRun` or
```

Comment:
There was an asterisk sign to show that the Command doesn't start with the `run` word.
Do you think we should omit that?

Reply:
It was confusing to me, I thought it's a formatting mistake

Reply:
Maybe we could think how not to confuse to our readers...